sormas-ui/src/main/java/de/symeda/sormas/ui/reports/aggregate/AggregateReportsView.java

```diff
@@ -74,19 +104,40 @@ public AggregateReportsView() {
 
 		addComponent(gridLayout);
 
+		OptionGroup aggregateReportViewSwitcher = new OptionGroup();
```

Comment:
I believe Martin's idea was to have a submenu as in the configurations or campaigns views, not an option group; @MartinWahnschaffe please confirm

Reply:
Yes, exactly

Reply:
updated